### PR TITLE
Pull request/cc84bca0

### PR DESCRIPTION
**Pull request by spencerlyon2**
Ref: https://github.com/spencerlyon2/CompEcon.jl/issues/26

### CODE REVIEW

**Comment by tkelman** [comment]
Does this need the matlab files to be checked into the repo? Could it be a submodule (if upstream uses git?) or link and not hurt anything?

**Comment by spencerlyon2** [comment]
Hey @tkelman sorry for the delay, I'm just seeing this message. It probably doesn't need matlab files in the repo. I've removed them. and updated this PR. 

@davidanthoff when I removed the matlab files I did a force push to the remote, git will complain about the remote and your local having diverged. It's probably best to just delete your local copy and re-clone.

File: CompEcon/versions/0.1.0/sha1 [hunk 1]
```diff
@@ -0,0 +1 @@
+c2f91cfd4e43c6a9768d15b49d3e0b2da22e02ac
```
Comment (tkelman) [inline CompEcon/versions/0.1.0/sha1]: can you push the git tag for this? https://github.com/spencerlyon2/CompEcon.jl/releases

Comment (spencerlyon2) [inline CompEcon/versions/0.1.0/sha1]: Yep should be done now.

**Pull request merged**